flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -129,6 +129,10 @@
 		.defaultValue("/opt/flink/log")
 		.withDescription("The directory that logs of jobmanager and taskmanager be saved in the pod.");
 
+	public static final String KUBERNETES_JOBMANAGER_ANNOTATION_PREFIX = "kubernetes.jobmanager.annotation.";
```

Comment:
The configuration in the document will not contain this config options.

https://ci.apache.org/projects/flink/flink-docs-master/ops/config.html

Reply:
1. Since the value can contain `:` and `,`, we should not use the map type.
2. We could add descriptions too in the document, just like what we have done for the `containerized.master.env.`

Reply:
I also notice that [FLINK-15640](https://issues.apache.org/jira/browse/FLINK-15640) plans to use the map config option.

Reply:
I think the map `ConfigOption` type supports to escape keys and values. Hence, if the user would need to define a value with `:` he could write it: `kubernetes.taskmanager.annotations: key1:'value:foobar','key:2':value`, for example. From that perspective I think it would indeed be better to use a map config option as @wangyang0918 suggested.

Reply:
Thanks for the explanation!  It makes sense to me, will change it to the map type.

Reply:
Yes, the key and value of map could be escaped. 
@zhengcanbin  FLINK-15640 will keep the same behavior with this ticket with using map type.

Reply:
That would be nice, let's do it.